### PR TITLE
Added NGINX Amplify

### PR DESCRIPTION
**Pull request by akamoroz**

### CODE REVIEW

File: data/tools/nginx-amplify.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "slug": "monitoring",
+    "name": "NGINX Amplify",
+    "description": "Amplify is free monitoring tool for NGINX. Amplify offers extended NGINX metric collection, visualization, monitoring, and a static analyzer for NGINX configuration. It's simple to set up, yet powerful enough to bring the much needed insight into NGINX performance. Amplify is a SaaS developed and operated by Nginx, Inc. Keep NGINX up to date and secure with a native monitoring solution.",
+    "url": "https://amplify.nginx.com",
+    "tags": [
+      "linux",
+      "bsd",
+      "free",
+      "commercial",
+      "ci", "cd",
```
Comment (rdsubhas) [inline data/tools/nginx-amplify.json]: I think ci/cd tags not applicable as it seems to be runtime monitoring

File: data/tools/nginx-amplify.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      "virt",
```
Comment (rdsubhas) [inline data/tools/nginx-amplify.json]: And I think `virt` also not applicable as its not a virtualization tool

File: data/tools/nginx-amplify.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      "cloud", "cloud-paas", "paas",
```
Comment (rdsubhas) [inline data/tools/nginx-amplify.json]: And I believe cloud/paas tags not applicable as its not a paas platform (we use that for platforms like heroku/etc)

File: data/tools/nginx-amplify.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      "logging", "monitoring",
```
Comment (rdsubhas) [inline data/tools/nginx-amplify.json]: logging/monitoring/metrics/visualization apply 👍

Comment (akamoroz) [inline data/tools/nginx-amplify.json]: @rdsubhas I'm Product Marketing Manager in Amplify. Apologies, the landing page is due to be updated very soon and we are going out of beta shortly. We already provide stable service to our customers. Beta status means only marketing/financial issues, not a product quality.

I thought "all relevant tags apply" (as in "monitoring virt servers", or "using static analyzer for ci/cd"). Anyway, you can change the list of tags (remove redundant ones).

URL (amplify.nginx.com) is only one thing which I ask to keep – we don't like to use nginx.com/amplify in any external places like your list.

Also, you can read our grateful documentation which located here: https://amplify.nginx.com/docs/

**Comment by akamoroz** [comment]
@rdsubhas I'm Product Marketing Manager in Amplify. Apologies, the landing page is due to be updated very soon and we are going out of beta shortly. We already provide stable service to our customers. Beta status means only marketing/financial issues, not a product quality.

I thought "all relevant tags apply" (as in "monitoring virt servers", or "using static analyzer for ci/cd"). Anyway, you can change the list of tags (remove redundant ones).

URL (amplify.nginx.com) is only one thing which I ask to keep – we don't like to use nginx.com/amplify in any external places like your list.

Also, you can read our grateful documentation which located here: https://amplify.nginx.com/docs/

**Comment by rdsubhas** [comment]
@akamoroz Great to catch up with you! Btw, are reviews are more to get your inputs 😄 Thanks for the clarification and I think we can merge this if the landing pages are the correct ones 👍 Merging...

**Pull request merged**